app/controllers/concerns/attachments_copier.rb

```diff
@@ -1,18 +1,19 @@
 module AttachmentsCopier
   def copy_attachments(record)
-    record.content.scan(Attachment::SCREENSHOT_REGEX).each do |screenshot_url|
-      _, _, _, _, project_id, node_id, filename, _ = screenshot_url
+    record.content.scan(Attachment::SCREENSHOT_REGEX).each do |screenshot_path|
+      full_screenshot_path, _, _, _, project_id, node_id, filename, _ = screenshot_path
 
-      attachment = Attachment.find_by(filename: filename, node_id: record.node_id_was)
+      attachment = Attachment.find_by(filename: CGI::unescape(filename), node_id: record.node_id_was)
 
       if attachment
         new_attachment = attachment.copy_to(record.node)
-        new_url = screenshot_url[0].gsub(
-          /nodes\/[0-9]+\/attachments/,
-          "nodes/#{new_attachment.node_id}/attachments"
+        new_filename = ERB::Util.url_encode(new_attachment.filename)
```

Comment:
Can we use [attachment.url_encoded_filename](https://github.com/secroots/dradis-pro/blob/2d14058431f2760904edbb2d74bfb83d4c7a12b8/app/models/attachment.rb#L238) here?